### PR TITLE
[advanced-reboot] Improve warm/fast-reboot timing data report

### PR DESCRIPTION
**Pull request by vaibhavhd**
### Description of PR

Summary: Reboot test summary fixes and improvements.

Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### How did you do it?
Improved regexes and restructured the summary report.

#### How did you verify/test it?
Tested on physical and KVM testbeds:

**PHYSICAL WARM:**
```
{
    "offset_from_kexec": {
        "database": "20.233144", 
        "finalizer": "20.259903", 
        "init_view": "39.397932", 
        "syncd_create_switch": "37.043072", 
        "port_init": "46.968407", 
        "port_ready": "47.116243", 
        "sai_create_switch": "47.866821", 
        "neighbor_entry": "51.063888", 
        "default_route_set": "52.990673", 
        "apply_view": "57.481362", 
        "lag_ready": "105.640816"
    }, 
    "controlplane": {
        "arp_ping": "", 
        "downtime": 68.464424
    }, 
    "time_span": {
        "radv": "83.983095", 
        "bgp": "69.992424", 
        "syncd": "35.879624", 
        "swss": "59.662855", 
        "teamd": "51.65728", 
        "database": "22.202607", 
        "syncd_create_switch": "10.42129", 
        "sai_create_switch": "0.001471", 
        "apply_view": "6.829703", 
        "init_view": "8.463846", 
        "neighbor_entry": "0.90703", 
        "port_init": "0.095757", 
        "port_ready": "0.289411", 
        "finalizer": "146.022336"
    }, 
    "reboot_type": "warm", 
    "dataplane": {
        "lost_packets": "0", 
        "downtime": 0.0
    }
}
```
**PHYSICAL FAST:**
```
{
    "offset_from_kexec": {
        "database": "26.71325", 
        "finalizer": "26.739669", 
        "init_view": "44.567428", 
        "syncd_create_switch": "", 
        "port_init": "58.402036", 
        "port_ready": "58.736259", 
        "sai_create_switch": "48.479516", 
        "neighbor_entry": "63.777894", 
        "default_route_set": "71.650744", 
        "apply_view": "58.363139", 
        "lag_ready": "44.242952"
    }, 
    "controlplane": {
        "arp_ping": "", 
        "downtime": 88.588592
    }, 
    "time_span": {
        "radv": "76.290094", 
        "bgp": "60.888731", 
        "syncd": "45.514229", 
        "swss": "52.814715", 
        "teamd": "64.043183", 
        "database": "30.30104", 
        "syncd_create_switch": "", 
        "sai_create_switch": "9.237282", 
        "apply_view": "0.001097", 
        "init_view": "3.907011", 
        "neighbor_entry": "7.968511", 
        "port_init": "1.817561", 
        "port_ready": "1.459397", 
        "finalizer": "1.236638"
    }, 
    "reboot_type": "fast", 
    "dataplane": {
        "lost_packets": "", 
        "downtime": 24.792314
    }
}
```

**KVM WARM:**
```
{
    "offset_from_kexec": {
        "database": "35.229929", 
        "finalizer": "35.245289", 
        "init_view": "47.015896", 
        "syncd_create_switch": "45.866473", 
        "port_init": "46.074554", 
        "port_ready": "48.447136", 
        "sai_create_switch": "49.638036", 
        "neighbor_entry": "54.967063", 
        "default_route_set": "58.691555", 
        "apply_view": "70.449995", 
        "lag_ready": "116.485555"
    }, 
    "controlplane": {
        "arp_ping": "", 
        "downtime": "98.662218"
    }, 
    "time_span": {
        "radv": "84.465778", 
        "bgp": "75.249058", 
        "syncd": "46.85737", 
        "swss": "65.388899", 
        "teamd": "57.501164", 
        "database": "37.048091", 
        "syncd_create_switch": "3.464588", 
        "sai_create_switch": "0.001063", 
        "apply_view": "9.935372", 
        "init_view": "2.619032", 
        "neighbor_entry": "2.688489", 
        "port_init": "5.539243", 
        "port_ready": "3.185543", 
        "finalizer": "137.2283"
    }, 
    "reboot_type": "warm", 
    "dataplane": {
        "lost_packets": "", 
        "downtime": 137.238094
    }
}
```

### CODE REVIEW

**Review by yxieca — APPROVED** [review]

File: tests/platform_tests/raw-messages [hunk 1]
```diff
@@ -0,0 +1,636 @@
+
```
Comment (shi-su) [inline tests/platform_tests/raw-messages:1]: Is this file necessary?

Comment (vaibhavhd) [reply tests/platform_tests/raw-messages]: Removed it now. I added to test the test - reduce time taken to run the entire test by directly jumping to log_analyzer.
But, this isn't necessarily useful.

**Review by shi-su — APPROVED** [review]
